assets/src/edit-story/app/media/utils/useMediaUploadQueue/index.js

```diff
@@ -55,8 +57,11 @@ function useMediaUploadQueue() {
     canTranscodeFile,
     transcodeVideo,
     getFirstFrameOfVideo,
+    convertGifToVideo,
   } = useFFmpeg();
 
+  const enableGifOptimization = useFeature('enableGifOptimization');
```

Comment:
This makes it appear as if it's a function. Should rename to `isGifOptimizationEnabled` to make it clearer it's a bool flag.

Reply:
Changed variable name.

Reply:
Apologies, I was not super clear on this one.

I meant changing the variable name, not the experiment name. Most of the feature flag names start with `enable`, so that's OK.

```js
const isGifOptimizationEnabled = useFeature('enableGifOptimization');
```

Reply:
Changed it back.